packages/web/components/timeline/TimelineMessage.tsx

```
@@ -1,347 +1,296 @@
-            <GoogleDocChatMessage message={message} />
+        </div>
+      );
+
+    // System notification
+    case 'SYSTEM_NOTIFICATION':
+      const severityColors = {
+        info: 'info',
+        warning: 'warning',
+        error: 'error',
+      };
+      const color = severityColors[event.data.severity] || 'info';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)